### ISSUE TITLE
mirrorbits loses ability to use GeoIP after GeoLite update

### ISSUE THREAD

**Issue by stormi** [etix/mirrorbits#93]
Each time I update the `GeoLite2-City.mmdb` and `GeoLite2-ASN.mmdb` files, mirrorbits apparently immediately loses the ability to read them. Any subsequent `?mirrorlist` request says `We were not able to use your IP to approximate your location, so have chosen the mirrors at random.`. 

Running `mirrorbits reload` solves it.

**Comment by etix** [comment]
Replacing the files on disk is enough to trigger the issue?

**Comment by stormi** [comment]
Yes, I simply copy the new files over the ones that are being used.
